static/app/views/acceptOrganizationInvite.spec.jsx

```diff
@@ -25,28 +28,25 @@ describe('AcceptOrganizationInvite', function () {
       existingMember: false,
     });
 
-    const wrapper = mountWithTheme(
-      <AcceptOrganizationInvite params={{memberId: '1', token: 'abc'}} />
-    );
-
-    const joinButton = wrapper.find('Button[aria-label="join-organization"]');
+    render(<AcceptOrganizationInvite params={{memberId: '1', token: 'abc'}} />);
 
-    expect(joinButton.exists()).toBe(true);
-    expect(joinButton.text()).toBe('Join the test-org organization');
+    const joinButton = getJoinButton();
+    expect(joinButton).toBeInTheDocument();
```

Comment:
`queryBy` is usually used when we want to check if an element is not rendered in the UI and it looks like we would like to check the opposite  

from the docs

> queryBy...: Returns the matching node for a query, and return null if no elements match. This is useful for asserting an element that is not present. Throws an error if more than one match is found (use queryAllBy instead if this is OK).

Reply:
I'm using `queryBy` in the helper since sometimes we DO want to check if it's not there using this helper

Reply:
yes, I noticed, but do we want to use it in both cases?

Reply:
I think it's fine. I'm sort of indifferent. We can duplicate the selectors but 🤷‍♂️